algorithms/kernel/kmeans/kmeans_csr_lloyd_distr_step1_fpt_dispatcher.cpp

```diff
@@ -28,6 +28,33 @@ namespace daal
 {
 namespace algorithms
 {
-__DAAL_INSTANTIATE_DISPATCH_CONTAINER(kmeans::DistributedContainer, distributed, step1Local, DAAL_FPTYPE, kmeans::lloydCSR)
+__DAAL_INSTANTIATE_DISPATCH_CONTAINER(kmeans::interface2::DistributedContainer, distributed, step1Local, DAAL_FPTYPE, kmeans::lloydCSR)
```

Comment:
no need to explicitly specify `kmeans::interface2`, using should be added in header file for interface2.

Reply:
I agree, that’s true. But if we add a new interface, transferring these files to the inner will be easier. Less likely to make errors with the name of the interfaces. Yes, and in principle I do not see anything bad in these cases, writing the full path to namespace.

I see a similar situation with new interfaces in the following algorithms: 
```
__DAAL_INSTANTIATE_DISPATCH_CONTAINER_SYCL(optimization_solver::cross_entropy_loss::interface2::BatchContainer, batch, DAAL_FPTYPE,
                                           optimization_solver::cross_entropy_loss::defaultDense)

__DAAL_INSTANTIATE_DISPATCH_CONTAINER_SYCL(optimization_solver::logistic_loss::interface2::BatchContainer, batch, DAAL_FPTYPE,
                                           optimization_solver::logistic_loss::defaultDense)

__DAAL_INSTANTIATE_DISPATCH_CONTAINER_SYCL(optimization_solver::mse::interface2::BatchContainer, batch, DAAL_FPTYPE,
                                           optimization_solver::mse::defaultDense)

__DAAL_INSTANTIATE_DISPATCH_CONTAINER(normalization::zscore::interface3::BatchContainer, batch, DAAL_FPTYPE, normalization::zscore::sumDense)

__DAAL_INSTANTIATE_DISPATCH_CONTAINER_SYCL(pca::interface3::BatchContainer, batch, DAAL_FPTYPE, pca::correlationDense)

__DAAL_INSTANTIATE_DISPATCH_CONTAINER(multinomial_naive_bayes::training::interface1::OnlineContainer, online, DAAL_FPTYPE,
                                      multinomial_naive_bayes::training::fastCSR)
```
Can we formulate a common solution on this?